12:17:19,663 (main) WARN  [DataNucleus.Datastore] - Default type for java type of java.math.BigInteger was previously jdbc-type=NUMERIC but this is not provided by the JDBC driver! Please report this to the DataNucleus developers
12:17:20,164 (main) ERROR [DataNucleus.Datastore.Schema] - Una excepcion estaba tirada durante la addicion/validacion de algunas class(es) : La generación del nuevo tipo de Mapping org.datanucleus.store.rdbms.mapping.java.CollectionMapping ha fallada : Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
org.datanucleus.exceptions.NucleusUserException: Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
	at org.datanucleus.metadata.AbstractMemberMetaData.setRelation(AbstractMemberMetaData.java:2738)
	at org.datanucleus.metadata.AbstractMemberMetaData.getRelatedMemberMetaData(AbstractMemberMetaData.java:3000)
	at org.datanucleus.store.rdbms.identifier.DNIdentifierFactory.newTableIdentifier(DNIdentifierFactory.java:133)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.addJoinTableForContainer(RDBMSStoreManager.java:3817)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.newJoinTable(RDBMSStoreManager.java:2966)
	at org.datanucleus.store.rdbms.mapping.java.AbstractContainerMapping.initialize(AbstractContainerMapping.java:84)
	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:667)
	at org.datanucleus.store.rdbms.table.ClassTable.manageMembers(ClassTable.java:632)
	at org.datanucleus.store.rdbms.table.ClassTable.manageClass(ClassTable.java:538)
	at org.datanucleus.store.rdbms.table.ClassTable.initializeForClass(ClassTable.java:1364)
	at org.datanucleus.store.rdbms.table.ClassTable.initialize(ClassTable.java:270)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.initializeClassTables(RDBMSStoreManager.java:3467)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.run(RDBMSStoreManager.java:3087)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.createSchemaForClasses(RDBMSStoreManager.java:3940)
	at org.datanucleus.store.schema.SchemaTool.createSchemaForClasses(SchemaTool.java:508)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:424)

La generación del nuevo tipo de Mapping org.datanucleus.store.rdbms.mapping.java.CollectionMapping ha fallada : Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
org.datanucleus.exceptions.NucleusUserException: Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
	at org.datanucleus.metadata.AbstractMemberMetaData.setRelation(AbstractMemberMetaData.java:2738)
	at org.datanucleus.metadata.AbstractMemberMetaData.getRelatedMemberMetaData(AbstractMemberMetaData.java:3000)
	at org.datanucleus.store.rdbms.identifier.DNIdentifierFactory.newTableIdentifier(DNIdentifierFactory.java:133)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.addJoinTableForContainer(RDBMSStoreManager.java:3817)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.newJoinTable(RDBMSStoreManager.java:2966)
	at org.datanucleus.store.rdbms.mapping.java.AbstractContainerMapping.initialize(AbstractContainerMapping.java:84)
	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:667)
	at org.datanucleus.store.rdbms.table.ClassTable.manageMembers(ClassTable.java:632)
	at org.datanucleus.store.rdbms.table.ClassTable.manageClass(ClassTable.java:538)
	at org.datanucleus.store.rdbms.table.ClassTable.initializeForClass(ClassTable.java:1364)
	at org.datanucleus.store.rdbms.table.ClassTable.initialize(ClassTable.java:270)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.initializeClassTables(RDBMSStoreManager.java:3467)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.run(RDBMSStoreManager.java:3087)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.createSchemaForClasses(RDBMSStoreManager.java:3940)
	at org.datanucleus.store.schema.SchemaTool.createSchemaForClasses(SchemaTool.java:508)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:424)

org.datanucleus.exceptions.NucleusException: La generación del nuevo tipo de Mapping org.datanucleus.store.rdbms.mapping.java.CollectionMapping ha fallada : Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
org.datanucleus.exceptions.NucleusUserException: Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
	at org.datanucleus.metadata.AbstractMemberMetaData.setRelation(AbstractMemberMetaData.java:2738)
	at org.datanucleus.metadata.AbstractMemberMetaData.getRelatedMemberMetaData(AbstractMemberMetaData.java:3000)
	at org.datanucleus.store.rdbms.identifier.DNIdentifierFactory.newTableIdentifier(DNIdentifierFactory.java:133)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.addJoinTableForContainer(RDBMSStoreManager.java:3817)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.newJoinTable(RDBMSStoreManager.java:2966)
	at org.datanucleus.store.rdbms.mapping.java.AbstractContainerMapping.initialize(AbstractContainerMapping.java:84)
	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:667)
	at org.datanucleus.store.rdbms.table.ClassTable.manageMembers(ClassTable.java:632)
	at org.datanucleus.store.rdbms.table.ClassTable.manageClass(ClassTable.java:538)
	at org.datanucleus.store.rdbms.table.ClassTable.initializeForClass(ClassTable.java:1364)
	at org.datanucleus.store.rdbms.table.ClassTable.initialize(ClassTable.java:270)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.initializeClassTables(RDBMSStoreManager.java:3467)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.run(RDBMSStoreManager.java:3087)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.createSchemaForClasses(RDBMSStoreManager.java:3940)
	at org.datanucleus.store.schema.SchemaTool.createSchemaForClasses(SchemaTool.java:508)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:424)

	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:677)
	at org.datanucleus.store.rdbms.table.ClassTable.manageMembers(ClassTable.java:632)
	at org.datanucleus.store.rdbms.table.ClassTable.manageClass(ClassTable.java:538)
	at org.datanucleus.store.rdbms.table.ClassTable.initializeForClass(ClassTable.java:1364)
	at org.datanucleus.store.rdbms.table.ClassTable.initialize(ClassTable.java:270)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.initializeClassTables(RDBMSStoreManager.java:3467)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.run(RDBMSStoreManager.java:3087)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.createSchemaForClasses(RDBMSStoreManager.java:3940)
	at org.datanucleus.store.schema.SchemaTool.createSchemaForClasses(SchemaTool.java:508)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:424)
Caused by: org.datanucleus.exceptions.NucleusUserException: Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
	at org.datanucleus.metadata.AbstractMemberMetaData.setRelation(AbstractMemberMetaData.java:2738)
	at org.datanucleus.metadata.AbstractMemberMetaData.getRelatedMemberMetaData(AbstractMemberMetaData.java:3000)
	at org.datanucleus.store.rdbms.identifier.DNIdentifierFactory.newTableIdentifier(DNIdentifierFactory.java:133)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.addJoinTableForContainer(RDBMSStoreManager.java:3817)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.newJoinTable(RDBMSStoreManager.java:2966)
	at org.datanucleus.store.rdbms.mapping.java.AbstractContainerMapping.initialize(AbstractContainerMapping.java:84)
	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:667)
	... 10 more
Nested Throwables StackTrace:
Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
org.datanucleus.exceptions.NucleusUserException: Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
	at org.datanucleus.metadata.AbstractMemberMetaData.setRelation(AbstractMemberMetaData.java:2738)
	at org.datanucleus.metadata.AbstractMemberMetaData.getRelatedMemberMetaData(AbstractMemberMetaData.java:3000)
	at org.datanucleus.store.rdbms.identifier.DNIdentifierFactory.newTableIdentifier(DNIdentifierFactory.java:133)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.addJoinTableForContainer(RDBMSStoreManager.java:3817)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.newJoinTable(RDBMSStoreManager.java:2966)
	at org.datanucleus.store.rdbms.mapping.java.AbstractContainerMapping.initialize(AbstractContainerMapping.java:84)
	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:667)
	at org.datanucleus.store.rdbms.table.ClassTable.manageMembers(ClassTable.java:632)
	at org.datanucleus.store.rdbms.table.ClassTable.manageClass(ClassTable.java:538)
	at org.datanucleus.store.rdbms.table.ClassTable.initializeForClass(ClassTable.java:1364)
	at org.datanucleus.store.rdbms.table.ClassTable.initialize(ClassTable.java:270)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.initializeClassTables(RDBMSStoreManager.java:3467)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.run(RDBMSStoreManager.java:3087)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.createSchemaForClasses(RDBMSStoreManager.java:3940)
	at org.datanucleus.store.schema.SchemaTool.createSchemaForClasses(SchemaTool.java:508)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:424)

12:17:20,165 (main) ERROR [DataNucleus.SchemaTool] - DataNucleus SchemaTool ha lanzado una excepción. Por favor consulte el Log para ver más detalles: La generación del nuevo tipo de Mapping org.datanucleus.store.rdbms.mapping.java.CollectionMapping ha fallada : Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
org.datanucleus.exceptions.NucleusUserException: Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
	at org.datanucleus.metadata.AbstractMemberMetaData.setRelation(AbstractMemberMetaData.java:2738)
	at org.datanucleus.metadata.AbstractMemberMetaData.getRelatedMemberMetaData(AbstractMemberMetaData.java:3000)
	at org.datanucleus.store.rdbms.identifier.DNIdentifierFactory.newTableIdentifier(DNIdentifierFactory.java:133)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.addJoinTableForContainer(RDBMSStoreManager.java:3817)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.newJoinTable(RDBMSStoreManager.java:2966)
	at org.datanucleus.store.rdbms.mapping.java.AbstractContainerMapping.initialize(AbstractContainerMapping.java:84)
	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:667)
	at org.datanucleus.store.rdbms.table.ClassTable.manageMembers(ClassTable.java:632)
	at org.datanucleus.store.rdbms.table.ClassTable.manageClass(ClassTable.java:538)
	at org.datanucleus.store.rdbms.table.ClassTable.initializeForClass(ClassTable.java:1364)
	at org.datanucleus.store.rdbms.table.ClassTable.initialize(ClassTable.java:270)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.initializeClassTables(RDBMSStoreManager.java:3467)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.run(RDBMSStoreManager.java:3087)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.createSchemaForClasses(RDBMSStoreManager.java:3940)
	at org.datanucleus.store.schema.SchemaTool.createSchemaForClasses(SchemaTool.java:508)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:424)

La generación del nuevo tipo de Mapping org.datanucleus.store.rdbms.mapping.java.CollectionMapping ha fallada : Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
org.datanucleus.exceptions.NucleusUserException: Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
	at org.datanucleus.metadata.AbstractMemberMetaData.setRelation(AbstractMemberMetaData.java:2738)
	at org.datanucleus.metadata.AbstractMemberMetaData.getRelatedMemberMetaData(AbstractMemberMetaData.java:3000)
	at org.datanucleus.store.rdbms.identifier.DNIdentifierFactory.newTableIdentifier(DNIdentifierFactory.java:133)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.addJoinTableForContainer(RDBMSStoreManager.java:3817)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.newJoinTable(RDBMSStoreManager.java:2966)
	at org.datanucleus.store.rdbms.mapping.java.AbstractContainerMapping.initialize(AbstractContainerMapping.java:84)
	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:667)
	at org.datanucleus.store.rdbms.table.ClassTable.manageMembers(ClassTable.java:632)
	at org.datanucleus.store.rdbms.table.ClassTable.manageClass(ClassTable.java:538)
	at org.datanucleus.store.rdbms.table.ClassTable.initializeForClass(ClassTable.java:1364)
	at org.datanucleus.store.rdbms.table.ClassTable.initialize(ClassTable.java:270)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.initializeClassTables(RDBMSStoreManager.java:3467)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.run(RDBMSStoreManager.java:3087)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.createSchemaForClasses(RDBMSStoreManager.java:3940)
	at org.datanucleus.store.schema.SchemaTool.createSchemaForClasses(SchemaTool.java:508)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:424)

org.datanucleus.exceptions.NucleusException: La generación del nuevo tipo de Mapping org.datanucleus.store.rdbms.mapping.java.CollectionMapping ha fallada : Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
org.datanucleus.exceptions.NucleusUserException: Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
	at org.datanucleus.metadata.AbstractMemberMetaData.setRelation(AbstractMemberMetaData.java:2738)
	at org.datanucleus.metadata.AbstractMemberMetaData.getRelatedMemberMetaData(AbstractMemberMetaData.java:3000)
	at org.datanucleus.store.rdbms.identifier.DNIdentifierFactory.newTableIdentifier(DNIdentifierFactory.java:133)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.addJoinTableForContainer(RDBMSStoreManager.java:3817)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.newJoinTable(RDBMSStoreManager.java:2966)
	at org.datanucleus.store.rdbms.mapping.java.AbstractContainerMapping.initialize(AbstractContainerMapping.java:84)
	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:667)
	at org.datanucleus.store.rdbms.table.ClassTable.manageMembers(ClassTable.java:632)
	at org.datanucleus.store.rdbms.table.ClassTable.manageClass(ClassTable.java:538)
	at org.datanucleus.store.rdbms.table.ClassTable.initializeForClass(ClassTable.java:1364)
	at org.datanucleus.store.rdbms.table.ClassTable.initialize(ClassTable.java:270)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.initializeClassTables(RDBMSStoreManager.java:3467)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.run(RDBMSStoreManager.java:3087)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.createSchemaForClasses(RDBMSStoreManager.java:3940)
	at org.datanucleus.store.schema.SchemaTool.createSchemaForClasses(SchemaTool.java:508)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:424)

	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:677)
	at org.datanucleus.store.rdbms.table.ClassTable.manageMembers(ClassTable.java:632)
	at org.datanucleus.store.rdbms.table.ClassTable.manageClass(ClassTable.java:538)
	at org.datanucleus.store.rdbms.table.ClassTable.initializeForClass(ClassTable.java:1364)
	at org.datanucleus.store.rdbms.table.ClassTable.initialize(ClassTable.java:270)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.initializeClassTables(RDBMSStoreManager.java:3467)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.run(RDBMSStoreManager.java:3087)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.createSchemaForClasses(RDBMSStoreManager.java:3940)
	at org.datanucleus.store.schema.SchemaTool.createSchemaForClasses(SchemaTool.java:508)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:424)
Caused by: org.datanucleus.exceptions.NucleusUserException: Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
	at org.datanucleus.metadata.AbstractMemberMetaData.setRelation(AbstractMemberMetaData.java:2738)
	at org.datanucleus.metadata.AbstractMemberMetaData.getRelatedMemberMetaData(AbstractMemberMetaData.java:3000)
	at org.datanucleus.store.rdbms.identifier.DNIdentifierFactory.newTableIdentifier(DNIdentifierFactory.java:133)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.addJoinTableForContainer(RDBMSStoreManager.java:3817)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.newJoinTable(RDBMSStoreManager.java:2966)
	at org.datanucleus.store.rdbms.mapping.java.AbstractContainerMapping.initialize(AbstractContainerMapping.java:84)
	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:667)
	... 10 more
Nested Throwables StackTrace:
Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
org.datanucleus.exceptions.NucleusUserException: Clase "es.deusto.ingenieria.sd.strava.data.domain.Usuario" campo "entrenamientos" : tiene "mapped-by" de "usuario". Este campo no existe en el otro lado de la relación ("es.deusto.ingenieria.sd.strava.data.domain.Entrenamiento")!!
	at org.datanucleus.metadata.AbstractMemberMetaData.setRelation(AbstractMemberMetaData.java:2738)
	at org.datanucleus.metadata.AbstractMemberMetaData.getRelatedMemberMetaData(AbstractMemberMetaData.java:3000)
	at org.datanucleus.store.rdbms.identifier.DNIdentifierFactory.newTableIdentifier(DNIdentifierFactory.java:133)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.addJoinTableForContainer(RDBMSStoreManager.java:3817)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.newJoinTable(RDBMSStoreManager.java:2966)
	at org.datanucleus.store.rdbms.mapping.java.AbstractContainerMapping.initialize(AbstractContainerMapping.java:84)
	at org.datanucleus.store.rdbms.mapping.MappingManagerImpl.getMapping(MappingManagerImpl.java:667)
	at org.datanucleus.store.rdbms.table.ClassTable.manageMembers(ClassTable.java:632)
	at org.datanucleus.store.rdbms.table.ClassTable.manageClass(ClassTable.java:538)
	at org.datanucleus.store.rdbms.table.ClassTable.initializeForClass(ClassTable.java:1364)
	at org.datanucleus.store.rdbms.table.ClassTable.initialize(ClassTable.java:270)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.initializeClassTables(RDBMSStoreManager.java:3467)
	at org.datanucleus.store.rdbms.RDBMSStoreManager$ClassAdder.run(RDBMSStoreManager.java:3087)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.createSchemaForClasses(RDBMSStoreManager.java:3940)
	at org.datanucleus.store.schema.SchemaTool.createSchemaForClasses(SchemaTool.java:508)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:424)
